config

```diff
@@ -27,3 +25,4 @@ export RH_CHE_DOCKER_IMAGE_VERSION=${RH_CHE_DOCKER_IMAGE_VERSION:-${DEFAULT_RH_C
 export DOCKER_HUB_USER=${DOCKER_HUB_USER:-${RHCHEBOT_DOCKER_HUB_USER}}
 export DOCKER_HUB_PASSWORD=${DOCKER_HUB_PASSWORD:-${RHCHEBOT_DOCKER_HUB_PASSWORD}}
 export DOCKER_HUB_NAMESPACE=${DOCKER_HUB_NAMESPACE:-${RH_CHE_DOCKER_HUB_NAMESPACE}}
+export DOCKER_HUB_REGISTRY_PREFIX=${DOCKER_HUB_REGISTRY_PREFIX:-""}
```

Comment:
I have a RHEL CSB laptop, and the CSB docker setup has the RH registry as primary docker registry. So by default when tagging or pushing without the `docker.io/` registry prefix, these actions are done against the `registry.access.redhat.com` registry, and push fails with an authentication failure. In such a use-case, being able to explicitly set the registry when tagging / pulling in the scripts helps a lot.
But of course the default value fro this is empty, so that it doesn't change anything for people that don't suffer of this RHEL CSB docker preset.